src/pgduckdb_guc.cpp

```diff
@@ -143,6 +144,9 @@ InitGUC() {
 	/* pg_duckdb specific GUCs */
 	DefineCustomVariable("duckdb.force_execution", "Force queries to use DuckDB execution", &duckdb_force_execution);
 
+	DefineCustomVariable("duckdb.unsafe_allow_execution_inside_functions", "Allow DuckDB execution inside functions",
+	                     &duckdb_unsafe_allow_execution_inside_functions);
```

Comment:
Given that this can introduce crashes I think a superuser should be configuring this.  

```suggestion
	                     &duckdb_unsafe_allow_execution_inside_functions, 0, GUC_SUPERUSER_ONLY);
```